backend/api/app.py

```diff
@@ -51,11 +54,27 @@ def ready(self):
         if RUNSERVER:
             try:
                 check_external_services()
+                # Seed data in Redis cache
+                self.seed_redis_cache()
             except RuntimeError as error:
                 print('Startup checks failed. Not starting.')
                 print(error)
                 exit(-1)  # Django doesn't seem to do this automatically.
 
+    def seed_redis_cache(self):
+        """Seed initial data into Redis cache"""
+        try:
+            # Check if data has already been seeded to avoid duplicates
+            if not redis_cache.get('redis_seeded'):
```

Comment:
We have to ensure this flag is set on the LCFS side as well when it populates its live cache. Please just confirm that is the case.

Reply:
sure, will update LCFS repo as well.